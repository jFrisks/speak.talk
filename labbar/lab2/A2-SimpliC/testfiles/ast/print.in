int a(){
    print(3);
}