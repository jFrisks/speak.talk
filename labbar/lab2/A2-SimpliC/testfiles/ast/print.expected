Program
  List
    FunctionDecl ID="a"
      List
      Block
        List
          FunctionCallStmt
            FunctionCall ID="print"
              List
                Numeral NUMERAL="3"